cmd/xenia/web/web.go

```diff
@@ -4,50 +4,99 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
+	"log"
 	"net/http"
+	"os"
 
 	"github.com/ardanlabs/kit/cfg"
+	"github.com/coralproject/shelf/internal/platform/auth"
 	"github.com/spf13/cobra"
 )
 
 const (
-	cfgHost = "WEB_HOST"
-	cfgAuth = "WEB_AUTH"
+	// cfgWebHost is the url to the web host which includes the hostname, port,
+	// and scheme.
+	cfgWebHost = "WEB_HOST"
+
+	// cfgPlatformPrivateKey is the private key used to sign new requests to the
+	// downstream service layer.
+	cfgPlatformPrivateKey = "PLATFORM_PRIVATE_KEY"
 )
 
+var authSigner auth.Signer
+
+func init() {
+	if err := cfg.Init(cfg.EnvProvider{Namespace: "XENIA"}); err != nil {
+		log.Println("Unable to initialize configuration")
+		os.Exit(1)
+	}
+
+	platformPrivateKey, err := cfg.String(cfgPlatformPrivateKey)
+	if err != nil {
+		log.Printf("Downstream Auth : Disabled : %s\n", err.Error())
+		return
+	}
+
+	// If the platformPrivateKey is provided, then we should generate the token
+	// signing function to be used when composing requests down to the platform.
+	if platformPrivateKey != "" {
```

Comment:
Maybe this could be added above?  like 

    platformPrivateKey, err := cfg.String(cfgPlatformPrivateKey)    
    if err != nil || platformPrivateKey == "" {
        blah
    }

Then, you wouldn't need this else statement.